master_changes.md

```diff
@@ -172,7 +172,7 @@ users)
   * Add switch-invariant test [#4866 @rjbou]
   * opam root version: add local switch cases [#4763 @rjbou] [2.1.0~rc2 #4715]
   * opam root version: add reinit test casess [#4763 @rjbou] [2.1.0~rc2 #4750]
-  * Add & update env tests [#4861 #4841 @rjbou @dra27]
+  * Add & update env tests [#4861 #4841 #4974 @rjbou @dra27 @AltGr]
```

Comment:
Can you add reverts (remove or put a line on them), and add the engine updates ?